crates/bevy_ui/src/ui_node.rs

```diff
@@ -1844,22 +1844,69 @@ impl TargetCamera {
     }
 }
 
+#[derive(Component)]
+/// Marker used to identify default cameras, they will have priority over the [`PrimaryWindow`] camera.
+///
+/// This is useful if the [`PrimaryWindow`] has two cameras, one of them used
+/// just for debug purposes and the user wants a way to choose the default [`Camera`]
+/// without having to add a [`TargetCamera`] to the root node.
+///
+/// Another use is when the user wants the Ui to be in another window by default,
+/// all that is needed is to place this component on the camera
+///
+/// ```
+/// # use bevy_ui::prelude::*;
+/// # use bevy_ecs::prelude::Commands;
+/// # use bevy_render::camera::{Camera, RenderTarget};
+/// # use bevy_core_pipeline::prelude::Camera2dBundle;
+/// # use bevy_window::{Window, WindowRef};
+///
+/// fn spawn_camera(mut commands: Commands) {
+///     let another_window = commands.spawn(Window {
+///         title: String::from("Another window"),
+///         ..Default::default()
+///     }).id();
+///     commands.spawn((
+///         Camera2dBundle {
+///             camera: Camera {
+///                 target: RenderTarget::Window(WindowRef::Entity(another_window)),
+///                 ..Default::default()
+///             },
+///             ..Default::default()
+///         },
+///         // We add the Marker here so all Ui will spawn in
+///         // another window if no TargetCamera is specified
+///         UiDefaultCameraMarker
+///     ));
+/// }
+/// ```
+pub struct UiDefaultCameraMarker;
```

Comment:
I'm not really a fan of the "Marker" suffix here, but at the same time not having the "Marker" suffix will make this components name very similar to the `DefaultUiCamera` `SystemParam`'s name.

Reply:
I don't really like that name too. We should for now think in another name for `DefaultUiCamera` or `UiDefaultCameraMarker`, but I don't have many ideas on this aspect 🤔

Reply:
`IsDefaultUiCamera` is an option.

Reply:
`IsDefaultUiCamera` is a good option, for now, better than what we have.

Reply:
Yep, let's move forward with that for 0.13.